code/modules/mining/boulder_processing/brm.dm

```diff
@@ -0,0 +1,130 @@
+#define MANUAL_TELEPORT_SOUND 'sound/machines/mining/manual_teleport.ogg'
+#define AUTO_TELEPORT_SOUND 'sound/machines/mining/auto_teleport.ogg'
+
+/obj/machinery/bouldertech/brm
+	name = "boulder retrieval matrix"
+	desc = "A teleportation matrix used to retrieve boulders excavated by mining NODEs from ore vents."
+	icon_state = "brm"
+	circuit = /obj/item/circuitboard/machine/brm
+	usage_sound = MANUAL_TELEPORT_SOUND
+	/// Are we trying to actively collect boulders automatically?
+	var/toggled_on = FALSE
+	/// How long does it take to collect a boulder?
+	var/teleportation_time = 1.5 SECONDS
+	// Cooldown used for left click teleportation.
+	COOLDOWN_DECLARE(manual_teleport_cooldown)
+
+/obj/machinery/bouldertech/brm/Initialize(mapload)
+	. = ..()
+	STOP_PROCESSING(SSmachines, src) // Don't start processing until flipped on.
```

Comment:
There is a flag for this, set `processing_flags = START_PROCESSING_MANUALLY` and it won't process on init